### PR TITLE
update alpine to v3.20.1

### PR DESCRIPTION
**Pull request by Zoey2936**

### CODE REVIEW

File: community-containers/dlna/dlna.json
Comment (szaimen) [inline community-containers/dlna/dlna.json:1]: Why are the permissions changed here? Can you revert?

**Review by szaimen — COMMENTED** [review]
Thanks Zoey! :)
One commen below:

File: community-containers/dlna/dlna.json
Comment (Zoey2936) [reply community-containers/dlna/dlna.json:1]: I tried to revert but everytime I clone the repo the permissions are changed for some reason

Comment (szaimen) [reply community-containers/dlna/dlna.json:1]: hm weird... so should we keep this permissions then?

Comment (Zoey2936) [reply community-containers/dlna/dlna.json:1]: No idea, can you maybe try to revert them?

Comment (szaimen) [reply community-containers/dlna/dlna.json:1]: It is not that important, lets keep the permissions

File: Containers/docker-socket-proxy/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:latest
-FROM haproxy:2.9.7-alpine3.19
+FROM haproxy:2.9.7-alpine3.20
```
Comment (Zoey2936) [inline Containers/docker-socket-proxy/Dockerfile:2]:
```suggestion
FROM haproxy:3.0.2-alpine3.20
```

Comment (szaimen) [reply Containers/docker-socket-proxy/Dockerfile:2]: lets do that in a different pr

Comment (Zoey2936) [reply Containers/docker-socket-proxy/Dockerfile:2]: ok, not sure why there was no automatic PR for this

Comment (szaimen) [reply Containers/docker-socket-proxy/Dockerfile:2]: I think because there is no 3.0.2-alpine3.19 available...

Comment (szaimen) [reply Containers/docker-socket-proxy/Dockerfile:2]: https://github.com/nextcloud/all-in-one/pull/4860

**Review by szaimen — APPROVED** [review]

**Comment by szaimen** [comment]
This is now released with v9.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Pull request merged**